Sources/PopupBridge/NetworkClient.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+protocol Networkable {
+    func post<T: Encodable>(url: URL, body: T) async throws
+}
+
+final class NetworkClient: Networkable {
+    
+    // MARK: - Private Properties
+    
+    private let session: Sessionable
+    
+    // MARK: - Initializer
+    
+    init(session: Sessionable = URLSession.shared) {
```

Comment:
Thoughts on just hardcoding `URLSession.shared` here vs using the `Sessionable` protoocol? Not sure if it's needed for testing.